src/registrar/models/domain.py

```diff
@@ -338,9 +344,6 @@ def __str__(self) -> str:
         help_text="Very basic info about the lifecycle of this domain object",
     )
 
-    def isActive(self):
```

Comment:
Copy this logic instead into is_active. There should not have been two, but this is the accurate implementation. I am actually wonder, when this is added in will your tests just fail? While we said to stub it out initially, I wonder if this a bad idea on my part. And instead, during your tests it should just use a patch for testing. In fact with it stubbed for false only one scenario is tested and we should also check that if it is active it doesn't move forward. What are your thoughts? (Beside "grr alysia why are you making me make changes so late")

Reply:
I'm testing invalid transitions in the model tests, using patching and active domains.